src/main/java/org/alliancegenome/curation_api/services/helpers/diseaseAnnotations/DiseaseAnnotationCurie.java

```diff
@@ -55,7 +55,7 @@ public String getCurieID(
         return curie.getCurie();
     }
 
-    private static String getExperimentalConditionCurie(ExperimentalCondition cond) {
+    public static String getExperimentalConditionCurie(ExperimentalCondition cond) {
```

Comment:
Curious on this change? Does this now get called from a place that wasn't before?

Reply:
Yes, from the validator.  That way, when the fields get updated the compound unique_id also gets updated.